commercial/app/model/feeds/FeedReader.scala

```diff
@@ -33,7 +33,6 @@ class FeedReader(wsClient: WSClient) extends Logging {
       val requestHolder: WSRequest = {
         val unsignedRequestHolder: WSRequest = wsClient.url(request.url)
           .withQueryString(request.parameters.toSeq: _*)
-          .withRequestTimeout(request.timeout)
```

Comment:
Does the circuit breaker timeout perform in exactly the same way as the `ws` one?  I wouldn't have thought it has access to the request in order to kill the connection, but maybe I am wrong.